autotests/testkeyverification.cpp

```diff
@@ -44,15 +60,18 @@ private Q_SLOTS:
                     QVERIFY(aSession);
                     QVERIFY(aSession->sasEmojis() == session->sasEmojis());
                     session->sendMac();
+                    aSession->sendMac();
```

Comment:
Soooo, does it mean that success of previous test runs was due to the test missing some important validation? :) In other words - should this test also check that upon `&KeyVerificationSession::finished` the respective party has received the MAC?

Reply:
i don't think this change is particularly important, it was something i did while trying to figure out why the QPointer is nullptr before finding the actual reason

Reply:
yes, checking for the mac actually being received makes sense, i'll add that